### DISCUSSION TITLE
Azure Blob image URL

### DISCUSSION THREAD

**Discussion by ivanamar996** [ImageMagick/ImageMagick#6459] [Help]
### ImageMagick version

7.1.0-55

### Operating system, version and so on

Windows

### Description

Hi,

I am using image magick portable version and I am wondering is it possible to use image url stored on azure blob storage with public access? 

I am trying to run the command with blob URL and I am getting this error : 

Process finished with 1 code. Error: magick.exe: UnableToOpenFile 'https://stprnconfdevwesteurope01.blob.core.windows.net/vectormaps/MO8933-03_POS1_vectormap.tif': No such file or directory @ error/url.c/ReadURLImage/212.

Kind regards,
Ivana Markovic

**Comment by snibgo** [comment]
What was your command? This command works fine for me:
```
magick https://stprnconfdevwesteurope01.blob.core.windows.net/vectormaps/MO8933-03_POS1_vectormap.tif x.png
```
Perhaps there is a security barrier on your computer.

What is the text output from:
```
magick -list policy
```

**Reply by ivanamar996** [reply]
It is working for me as well when I run the command locally, but it fails when I deploy same code on azure app service.

This is command that I am trying to use: 

magick.exe C:\home\site\wwwroot\position-preview.png -resize "2695x2710^!" https://stprnconfdevwesteurope01.blob.core.windows.net/vectormaps/MO8933-03_POS1_vectormap.tif -virtual-pixel Edge -compose Distort -define -compose:args=100%%x100%% -composite C:\home\site\wwwroot\product.jpg -compose Overlay -composite C:\home\site\wwwroot\result.png

Here is result of _magick -list policy_ locally:

  Policy: System
    name: precision
    value: 6

Path: [built-in]
  Policy: Undefined
    rights: None

and this is what I get on azure: 

Path: [built-in]
  Policy: Undefined
    rights: None

**Reply by fmw42** [reply]
Check 

`magick -list resource`

on each system

To check your policy, do the following

`find /usr -name policy.xml`

You should then find something like

```
/usr/local/etc/ImageMagick-6/policy.xml
or
/usr/local/etc/ImageMagick-7/policy.xml
```

Once you get that, then use it to do a cat on that file, such as

`cat /usr/local/etc/ImageMagick-6/policy.xml`

**Comment by snibgo** [comment]
I know nothing about "azure app service." Perhaps there is a security wall. Does `wget` work?
